NLog.Targets.KafkaAppender/KafkaAppender.cs

```diff
@@ -34,6 +34,21 @@ public class KafkaAppender : TargetWithLayout
         /// </summary>
         public Layout SslCertificateLocation { get; set; }
 
+        /// <summary>
+        /// Path to CA (certificate authority) certificate used for authentication.
+        /// </summary>
+        public Layout SslCaLocation { get; set; }
+
+        /// <summary>
+        /// Path to certificate key used for authentication.
+        /// </summary>
+        public Layout SslKeyLocation { get; set; }
+
+        /// <summary>
+        /// SSL key password
+        /// </summary>
+        public string SslKeyPassword { get; set; }
```

Comment:
Any reason for not using `Layout` ?

Reply:
Not really. I'm not really familiar with the NLog customization, and just wanted to avoid call to `RenderLogEvent()` and `LogEventInfo.CreateNullEvent()`.

I agree with you, if `Layout` allows also env-variables then it's a better option.